Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -453,6 +489,18 @@ void EnhancementsWidget::AddDescriptions()
       "scaling filter selected by the game.<br><br>Any option except 'Default' will alter the look "
       "of the game's textures and might cause issues in a small number of "
       "games.<br><br><dolphin_emphasis>If unsure, select 'Default'.</dolphin_emphasis>");
+  static const char TR_OUTPUT_RESAMPLING_DESCRIPTION[] = QT_TR_NOOP(
+      "Affects how the game output image is upscaled or downscaled to the window resolution.<br>"
+      "\"Default\" will rely on the GPU internal bilinear sampler which isn't gamma corrected."
+      "<br>\"Bilinear\" (gamma corrected) is a good compromise between quality and performance."
+      "<br>\"Bicubic\" is smoother than \"Bilinear\"."
```

Comment:
"smoother" doesn't really mean anything here. In practice, bicubic is _slightly_ sharper than bilinear.

Honestly I'm not sure why you even are offering bicubic when it's not a substantial improvement over bilinear - it seems like an easy cut to simplify this drop down.